src/modules/articles/components/ArticleMedia.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { Gallery } from '../../media/components';
+import { Lightbox } from '../../core/components';
+import ArticleMedium from './ArticleMedium';
+
+import { IMedium, IArticle } from '../queries';
+
+interface IProps {
+    article: IArticle,
```

Comment:
I'm a little confused on why we're passing an article and the media. What purpose does each serve?

Reply:
The article is used for things like article title in the lightbox. The media parameter lets consumers select the media that will be rendered.